ambari-server/src/main/java/org/apache/ambari/server/configuration/Configuration.java

```diff
@@ -1866,14 +1866,28 @@
    */
   @Markdown(description = "The maximum size of an incoming stomp text message. Default is 2 MB.")
   public static final ConfigurationProperty<Integer> STOMP_MAX_INCOMING_MESSAGE_SIZE = new ConfigurationProperty<>(
-      "stomp.max.message.size", 2*1024*1024);
+      "stomp.max_incoming.message.size", 2*1024*1024);
 
   /**
    * The maximum size of a buffer for stomp message sending. Default is 5 MB.
    */
   @Markdown(description = "The maximum size of a buffer for stomp message sending. Default is 5 MB.")
   public static final ConfigurationProperty<Integer> STOMP_MAX_BUFFER_MESSAGE_SIZE = new ConfigurationProperty<>(
-      "stomp.max.message.size", 5*1024*1024);
+      "stomp.max_buffer.message.size", 5*1024*1024);
+
+  /**
+   * The number of attempts to emit execution command message to agent. Default is 4
+   */
+  @Markdown(description = "The number of attempts to emit execution command message to agent. Default is 4")
+  public static final ConfigurationProperty<Integer> EXECUTION_COMMANDS_RETRY_COUNT = new ConfigurationProperty<>(
+      "stomp.commands.retry.count", 4);
```

Comment:
Minor nit: I would've named this as execution.command.retry.count instead of the protocol as config prefix.